.storybook/recipes/BaselineCard/BaselineCard.tsx

```diff
@@ -30,6 +31,14 @@ export interface Props {
    * CSS class names that can be appended to the component.
    */
   className?: string;
+  /**
+   * Clickable card: link URL
+   */
+  linkHref?: string;
+  /**
+   * Clickable card: link text
+   */
```

Comment:
Maybe add that the link text will only be visible when focused by a keyboard.

Reply:
Nice, thank you.